### PR TITLE
replaced Start/Stop Buttons with Switches

### PR DESCRIPTION
**Pull request by rabbit-zero**

### CODE REVIEW

File: Apps/frontend/src/views/StartStopButton.svelte [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
 	export let channel_id;
 
 
 	const dispatch = createEventDispatcher()
 
-	let play = "Start ▶";
-	let stop = "Stop ■";
-	let symbol = channel_id + " " + stop;
 	let hasStarted = true;
+	let switchLabelOn = "Started " + channel_id;
```
Comment (leandertolksdorf) [inline Apps/frontend/src/views/StartStopButton.svelte:10]: Not sure if the label "Started/Stopped" is needed anymore, as the toggle buttons already show the current state.

File: Apps/frontend/src/views/Oscilloscope.svelte [hunk 2]
```diff
@@ -93,7 +93,7 @@
         />
       </div>
       <div class="slider-wrapper">
-        <div class="sliders">
+        <div class="switches">
           Start/Stop
           <br>
           {#each { length: NUM_CHANNELS } as _, index}
```
Comment (leandertolksdorf) [inline Apps/frontend/src/views/Oscilloscope.svelte:99]: When merging, please make sure to keep updating the indicators' start/stop state in the on:startStop function.

**Review by leandertolksdorf — CHANGES_REQUESTED** [review]
The toggle buttons are not working as intended on my machine (see video).

**Comment by leandertolksdorf** [comment]
https://user-images.githubusercontent.com/55246283/206021669-e15120fb-0e65-4ad2-a7b6-5425b58cf9eb.mov

File: Apps/frontend/src/views/StartStopButton.svelte [hunk 3]
```diff
@@ -1,34 +1,24 @@
...
 	}
 
 
 </script>
 
-<button class="button-style" on:click={handleStartStop}>
-	{symbol}
-</button>
+<Switch bind:value={hasStarted} hideLabel on={switchLabelOn}
+off={switchLabelOff} design="slider" on:click={handleStartStop} />
```
Comment (PhlppKrmr) [inline Apps/frontend/src/views/StartStopButton.svelte:22]: What is `design="slider"` supposed to do?

File: Apps/frontend/src/views/StartStopButton.svelte [hunk 4]
```diff
@@ -1,34 +1,24 @@
...
 		dispatch('startStop', {buttonValue: hasStarted})
 	}
 
 
 </script>
 
-<button class="button-style" on:click={handleStartStop}>
-	{symbol}
-</button>
+<Switch bind:value={hasStarted} hideLabel on={switchLabelOn}
```
Comment (PhlppKrmr) [inline Apps/frontend/src/views/StartStopButton.svelte:21]: You can look at what I did in #79 and change the color(s) of the the switch, maybe we can eben remove the label?

**Review by PhlppKrmr — CHANGES_REQUESTED** [review]
If you need something to orient yourself on look at #79, I think you can adopt quite a lot from it.

File: Apps/frontend/src/views/Oscilloscope.svelte [hunk 5]
```diff
@@ -94,7 +94,7 @@
         />
       </div>
       <div class="slider-wrapper">
-        <div class="sliders">
+        <div class="switches">
```
Comment (PhlppKrmr) [inline Apps/frontend/src/views/Oscilloscope.svelte:97]: We don't have this class. Change to `sliders`

**Review by PhlppKrmr — APPROVED** [review]
Rest looks good.

**Review by jenswaechtler — APPROVED** [review]

**Review by jandegen — APPROVED** [review]

**Review by PhlppKrmr — APPROVED** [review]

**Pull request merged**